MaterialLib/SolidModels/MFront/MohrCoulombAbboSloanAniso.mfront

```diff
@@ -0,0 +1,291 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+@DSL Implicit;
+@Behaviour MohrCoulombAbboSloanAniso;
+@Author Thomas Nagel;
+@Date 21 / 03 / 2020;
+
+@Algorithm NewtonRaphson;
+@MaximumNumberOfIterations 200;
+
+@Description
+{
+    Non -
```

Comment:
Is there a reason for the formatting of the description section?

Reply:
Clang-format doesn't recognize it as ordinary test I guess ... I manually undid the machine's changes ... uh ... ;-)